docs/examples/obspy/create_staxml.py

```diff
@@ -0,0 +1,46 @@
+from obspy.clients.nrl import NRL
+from obspy.core.inventory import Inventory, Network, Station, Channel, Site
+from obspy.core import UTCDateTime
+from obspy.io.stationxml.core import validate_stationxml
+
+def do_xml():
+    nrl = NRL('http://ds.iris.edu/NRL/')
+    datalogger_keys = ['REF TEK', 'RT 130 & 130-SMA', '1', '40']
+    sensor_keys = ['Streckeisen', 'STS-2', '1500', '3 - installed 04/97 to present']
+
+    response = nrl.get_response(sensor_keys=sensor_keys, datalogger_keys=datalogger_keys)
+
+    channel = Channel(code='BHZ',
+                      location_code='10',      # required
+                      latitude=0,      # required
+                      longitude=0,   # required
+                      elevation=0.0,        # required
+                      depth=0.,                # required
+                      )
+
+    channel.response = response
+    station = Station(code='ABCD',
+                      latitude=0,
+                      longitude=0,
+                      elevation=0.0,
+                      creation_date=UTCDateTime(1970, 1, 1),          # required
+                      site=Site(name='Fake Site'),  # required
+                      channels=[channel],
+                      )
+
+    network = Network(code='XX',
+                     stations=[station])
+    inventory = Inventory(networks=[network], source="demo")
+
+    inventory.write("Test.xml", format="stationxml", validate=False)
+    (is_valid, error_list) = validate_stationxml("Test.xml")
+    if is_valid:
+        print("Test.xml validation ok ")
+    else:
+        print("Errors in validation: ")
+        for err in error_list:
+            print(err)
```

Comment:
@crotwell, personally, if you show this as a best practice example, I would probably stay away from "print debugging".. just be safe, put `validate=True` there to force it to fail with an exception. If people don't realize they wanna look for that printed message it could get overlooked. You could add a comment `# set validate=False to produce output even if it is not strictly valid StationXML if needed`.

Or maybe you could change your example to..

```python
try:
    inventory.write("Test.xml", format="stationxml", validate=True)
except Exception:
    inventory.write("Test.xml", format="stationxml", validate=False)
    (is_valid, error_list) = validate_stationxml("Test.xml")
    print("Errors in validation: ")
    for err in error_list:
        print(err)
```

We have a tutorial snippet in our docs that you could link, although it isn't any more advanced than your example here, so that might not be needed.

Reply:
Good points, will change to just `validate=True` and delete the prints. Simple is best here.
Thanks for reviewing...

Reply:
:+1: